docs/ide-setup.md

```diff
@@ -1,11 +1,11 @@
-# Using an IDE
+# Set up an IDE
 
 - [Eclipse IDE](#eclipse-ide)
 - [IntelliJ IDEA](#intellij-idea)
 
 ## Eclipse IDE
 
-> - Replace all references of `Eclipse → Preferences → ...` to `Window → Preferences → ...` if you are using Windows or Linux.
+> - If you are using Windows, replace all references of `Eclipse → Preferences → ...` to `Window → Preferences → ...`. The steps are similar for Linux users.
```

Comment:
This is unnecessary. The previous version is more concise and clear enough.

Reply:
Yes. Sorry, I misread and misunderstood the initial version. Will change it back.